### PR TITLE
Implement management command to update email addresses on tickets.

### PR DESCRIPTION
**Pull request by mandan**

### CODE REVIEW

**Comment by mandan** [comment]
[[general-308] Changing emails made all my support tickets disappear (Opened)](https://trello.com/c/E6A4bOpZ/243-general-308-changing-emails-made-all-my-support-tickets-disappear-opened)

File: helpdesk/management/commands/change_emails.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError, ObjectDoesNotExist
```
Comment (abannerjee) [inline helpdesk/management/commands/change_emails.py]: Are 'validate_email' and 'ValidationError' used in this file?

File: helpdesk/management/commands/change_emails.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+from django.contrib.auth.models import User
+
+from helpdesk.models import Ticket
+
+class Command(BaseCommand):
```
Comment (abannerjee) [inline helpdesk/management/commands/change_emails.py:9]: Minor issue, should have 2 blank lines above class definition.

File: helpdesk/management/commands/change_emails.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+        try:
+            User.objects.get(email=new_email)
+        except ObjectDoesNotExist:
+            raise CommandError('%s is not valid for any user' % new_email)
+
+        old_tickets = Ticket.objects.filter(submitter_email=old_email)
+
+        self.stdout.write('%s has %i tickets' %
+            (old_email, len(old_tickets)))
```
Comment (abannerjee) [inline helpdesk/management/commands/change_emails.py]: This line (and others like it) should be indented again. See following flake8 errors:

```
helpdesk/management/commands/change_emails.py:32:13: E128 continuation line under-indented for visual indent
helpdesk/management/commands/change_emails.py:35:13: E128 continuation line under-indented for visual indent
helpdesk/management/commands/change_emails.py:44:14: E128 continuation line under-indented for visual indent
helpdesk/management/commands/change_emails.py:48:13: E128 continuation line under-indented for visual indent
helpdesk/management/commands/change_emails.py:50:13: E128 continuation line under-indented for visual indent
```

**Pull request merged**